decidim-core/app/helpers/decidim/external_domain_helper.rb

```diff
@@ -2,6 +2,7 @@
 
 module Decidim
   module ExternalDomainHelper
+    # deprecated
```

Comment:
Is this deprecated because of the redesign? In that case I'd mention redesign here